src/services/servers/servers.js

```diff
@@ -1,9 +1,7 @@
-import { getFromLocalStorage } from "../../utils/localStorage/localStorage";
-
 const url = "http://playground.tesonet.lt/v1/servers";
 
 const fetchServers = async () => {
-  const token = getFromLocalStorage("token");
+  const token = localStorage.getItem("token");
```

Comment:
Is tikro tai sitas daug kur naudojamas, tai butu gerai, kad "token" butu variable. Arba, kad butu panasiai kaip seniau buvo ` export const getTokenFromLocalStorage = () => localStorage.getItem("token");`  😄